Makefile

```diff
@@ -286,3 +290,11 @@ ci-install-hypershift-private:
 regenerate-pki:
 	REGENERATE_PKI=1 $(GO) test ./control-plane-pki-operator/...
 	REGENERATE_PKI=1 $(GO) test ./test/e2e/... -run TestRegeneratePKI
+
+.PHONY: hypershift-install-aws
+hypershift-install-aws:
```

Comment:
hypershift-install-aws-dev?

Reply:
Are you saying I should rename it to hypershift-install-aws-dev?